contribs/github-bot/client/client.go

```diff
@@ -0,0 +1,255 @@
+package client
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/gnolang/gno/contribs/github-bot/logger"
+	p "github.com/gnolang/gno/contribs/github-bot/params"
+
+	"github.com/google/go-github/v64/github"
+)
+
+const PageSize = 100
+
+type GitHub struct {
+	Client *github.Client
+	Ctx    context.Context
```

Comment:
Because unfortunately, the context is not attached to the go-github client, and you need to pass it as a parameter for each request. So every part of the code that needs to make a request needs to access this context.